lib/ramble/ramble/test/end_to_end.py

```diff
@@ -239,12 +239,14 @@ def test_wrfv4_dry_run():
 
         output = workspace('analyze', '-f', 'text',
                            'json', 'yaml', global_args=['-w', workspace_name])
+        text_simlink_results_files = glob.glob(os.path.join(ws1.root, 'results.latest.txt'))
```

Comment:
It might be nice to add a test that if `workspace analyze` is run multiple times, `results.latest.txt` still points to the (actual) latest file. Maybe we should add a task for tracking this.

Reply:
Good idea. I filed one onto our backlog